app/controllers/openid_connect/logout_controller.rb

```diff
@@ -41,5 +71,9 @@ def logout_params
       end
       params.permit(*permitted)
     end
+
+    def render_404_if_disabled
+      render_not_found unless IdentityConfig.store.accept_client_id_in_oidc_logout || IdentityConfig.store.reject_id_token_hint_in_logout
```

Comment:
👀 compound `unless`, my mortal enemy

```suggestion
     if !IdentityConfig.store.accept_client_id_in_oidc_logout &&
       !IdentityConfig.store.reject_id_token_hint_in_logout
       render_not_found
     end
```

Reply:
Rubocop was also not a fan and [Andrew's suggestion](https://github.com/18F/identity-idp/pull/7017#pullrequestreview-1122623993) means it's gone